packages/core/src/components/way-input/way-input.scss

```diff
@@ -0,0 +1,92 @@
+:host {
+  /**
+    * @prop --border-radius: Border radius of the input
+    * @prop --border-width: Border width of the button
+    * @prop --border-style: Border style of the button
+    * @prop --border-color: Border color of the button
+    */
+  display: inline-block;
+
+  width: auto;
+
+  font-family: var(--way-font-family, inherit);
+  font-size: var(--way-font-size);
+  line-height: 1.25rem;
+
+  user-select: none;
+  vertical-align: top;
+  vertical-align: -webkit-baseline-middle;
+  pointer-events: auto;
+
+  font-kerning: none;
+
+  --border-radius: 5px;
+  --border-width: 3px;
```

Comment:
**issue**: Wat mij betreft gewoon `1px` (evt wel een CSS var van maken in de core.scss), ook ivm consistentie.

Reply:
ook done